migration/src/m0000630_alter_qualified_purl_purl_column.rs

```diff
@@ -0,0 +1,109 @@
+use sea_orm_migration::prelude::*;
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        //create jsonb purl column
+        manager
+            .alter_table(
+                Table::alter()
+                    .table(QualifiedPurl::Table)
+                    .add_column(ColumnDef::new(QualifiedPurl::Purl).json_binary())
+                    .to_owned(),
+            )
+            .await?;
+
+        //set up indexes
+        manager
+            .get_connection()
+            .execute_unprepared(
+                r#"
+                CREATE INDEX QualifiedPurlNameJsonGistIdx ON qualified_purl USING GIST ((purl ->> 'name') gist_trgm_ops);
+                CREATE INDEX QualifiedPurlVersionJsonGistIdx ON qualified_purl USING GIST ((purl ->> 'version') gist_trgm_ops);
+                CREATE INDEX QualifiedPurlNamespaceJsonGistIdx ON qualified_purl USING GIST ((purl ->> 'namespace') gist_trgm_ops);
+                CREATE INDEX QualifiedPurlTypeJsonGistIdx ON qualified_purl USING GIST ((purl ->> 'ty') gist_trgm_ops);
+                "#,
+            )
+            .await?;
+
+        //data migration
+        let _data_migration_result = manager
```

Comment:
I think the `?` at the end counts as "using" the result, so you can avoid the assignment to a `_var`.